tap_closeio/streams.py

```diff
@@ -115,15 +118,24 @@ def paginated_sync(tap_stream_id, ctx, request, start_date):
             # There may be streams other than `leads` that will run into
             # `max_skip` errors but YAGNI. We can make the tap more
             # complicated once we have an extant need for it.
-            if 'max_skip = ' in str(e) and tap_stream_id == IDS.LEADS:
-                LOGGER.info(("Hit max_skip error. "
-                             "Setting bookmark to `{}` and restarting pagination.".format(
-                                 max_bookmark)))
-                skip = 0
-                ctx.clear_offsets(tap_stream_id)
-                ctx.set_bookmark(bookmark(tap_stream_id), max_bookmark)
-                _request = create_leads_request(ctx)
-                ctx.write_state()
+            if 'max_skip = ' in str(e):
+                if tap_stream_id == IDS.ACTIVITIES:
+                    LOGGER.warning("Hit max_skip error so clearing skip offset, please reduce the date window size and try again.")
+                    # clear offset
+                    ctx.clear_offsets(tap_stream_id)
+                    ctx.write_state()
+                    raise
```

Comment:
Need to add the message to reduce the date window.

Reply:
Updated error message